scripts/build.py

```diff
@@ -110,15 +110,19 @@
     'third_party/generated/webfonts/*',
     '*.bundle.js',
     '*.bundle.js.map',
+    'webpack_bundles/*'
```

Comment:
I see lots of files in webpack_bundels (Which are not served statically.) don't we need hash in those files as well?

Reply:
No, these are automatically hashed by webpack.

Reply:
I see, thanks for the info!